### PR TITLE
Security chapter

### PR DESCRIPTION
**Pull request by SaptakS**
Refs: #2150

### CODE REVIEW

**Comment by rviscomi** [comment]
@SaptakS mark this "ready for review" when ready

**Comment by SaptakS** [comment]
@rviscomi Yep. Just waiting on a few things. Should be ready for review by tonight hopefully.

**Review by tunetheweb — CHANGES_REQUESTED** [review]
A fantastic conversion!

I've drop a few comments (mostly editing nits)that will just make the subsequent copy edit PR a lot smaller.

@tomvangoethem @nrllh you happy to merge this?

File: src/content/en/2021/security.md [hunk 1]
```diff
@@ -1,20 +1,1186 @@
...
+
+{{ figure_markup(
+  caption='The percentage of requests that use HTTPS on mobile.<br>(Source: <a hreflang="en" href="https://httparchive.org/reports/state-of-the-web#pctHttps">HTTP Archive</a>)',
+  content="91.3%",
+  classes="big-number",
+  sheets_gid="8055510"
+)
+}}
+
+Currently, we see that 92.3% of total requests for websites on desktop and 91.3% of total requests for websites on mobile are being served using HTTPS. We see an [increasing number of certificates](https://letsencrypt.org/stats/#daily-issuance) being issued every day thanks to non-profit certificate authorities like Let's Encrypt.
```
[the remark this answers is not in the document]
Comment (SaptakS) [reply src/content/en/2021/security.md:44]: This is only for external links, right?

[the remark this answers is not in the document]
Comment (tunetheweb) [reply src/content/en/2021/security.md:44]: Correct. We remain optimistic that the internal links will all be translated (someday!).

Also it’s unnecessary (and preferred not to have) for Twitter links (they offer an auto translate option on Tweets), MDN (they have most pages translated to at least one language and we’ll show the `(en)` UI based on the `en-US` in the URL) and Wikipedia (same).

**Comment by SaptakS** [comment]
@tunetheweb addressed all the above comments.

**Comment by tunetheweb** [comment]
OK made a few more tweaks but think this is good to merge. There'll be a copy edit after this and then we're good.

Here's how it looks now: https://20211124t165615-dot-webalmanac.uk.r.appspot.com/en/2021/security

**Pull request merged**